driver-core/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     compile "io.netty:netty-buffer:$nettyVersion", optional
     compile "io.netty:netty-transport:$nettyVersion", optional
     compile "io.netty:netty-handler:$nettyVersion", optional
+    compile "org.xerial.snappy:snappy-java:1.1.4", optional
```

Comment:
Suggestion - keep the version numbers in one place: `mongo-java-driver/build.gradle`.